packages/commerce-sdk-react/src/hooks/ShopperExperience/query.ts

```diff
@@ -14,11 +14,13 @@ import * as queryKeyHelpers from './queryKeyHelpers'
 type Client = ApiClients['shopperExperience']
 
 /**
- * Get Page Designer pages. The results will apply the visibility rules for each page's components, such as personalization or scheduled visibility.
-
-Either `categoryId` or `productId` must be given in addition to `aspectTypeId`. Because only a single page-to-product and page-to-category assignment per aspect type can be authored today, the returned results contains one element at most.
-
-**Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
+ * Get Page Designer pages.
+ *
+ * @remarks
+ * The results will apply the visibility rules for each page's components, such as personalization or scheduled visibility. Either `categoryId` or `productId` must be given in addition to `aspectTypeId`. Because only a single page-to-product and page-to-category assignment per aspect type can be authored today, the returned results contains one element at most.
+ * *Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
```

Comment:
```suggestion
 * **Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
```